packages/web-app-admin-settings/tests/unit/views/Spaces.spec.ts

```diff
@@ -12,14 +12,6 @@ import {
 } from 'web-test-helpers'
 import Spaces from '../../../src/views/Spaces.vue'
 
-window.ResizeObserver =
-  window.ResizeObserver ||
-  jest.fn().mockImplementation(() => ({
-    disconnect: jest.fn(),
-    observe: jest.fn(),
-    unobserve: jest.fn()
-  }))
-
```

Comment:
Fyi this has been moved to `jest.init.ts` so we don't have to add this for a dozen of tests.